f600233d1b154d507b782f84604bcb12/update/sys_script_eacf8da0970d91102b1ef0b0f053af4e.xml

```diff
@@ -36,7 +36,7 @@
 	var aclObject = {
 		userSysId: gs.getUserID(),
 		userRoles: gs.getUser().getRoles().toString().split(","), 
-		accessList: current.primary_reviewer.toString().split(","),
+		accessList: current.primary_reviewer && current.primary_reviewer.user ? current.primary_reviewer.user.toString().split(",") : null,
```

Comment:
If it was intended to leave the underlying column name as `primary_reviewer` this is fine, but if not it would need to be updated.

Reply:
There were two changes - the name AND the type, from a `Reference` to `User` to a `Reference` to `DITCO Contract Specialist`.